Vagrantfile

```diff
@@ -133,8 +133,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       aws.access_key_id = ENV['AWS_ACCESS_KEY']
       aws.secret_access_key = ENV['AWS_SECRET_KEY']
       aws.keypair_name = keypair
-      aws.ami = "ami-e7ab8a9c" # Ubuntu Trusty LTS
-      aws.block_device_mapping = [{ 'DeviceName' => '/dev/sda1', 'Ebs.VolumeSize' => 500 }]
+      aws.ami = "ami-0f9cf087c1f27d9b1" # Ubuntu Server 16.04 LTS
```

Comment:
You indicate you are using an Ubuntu 16.04 (Xenial) AMI for AWS but you still use Ubuntu 14.04 (Trusty) for the other providers (VirtualBox and OpenStack on Chameleon Cloud).  You should use Xenial throughout, i.e., use `override.vm.box = "ubuntu/trusty64"`  in the `:virtualbox` provider and use `os.image  = "CC-Ubuntu16.04" (I believe that's the correct image name) in the `: openstack` provider.